general/contributing/sourcetree.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: contrib-source
+title: Source Tree
+---
+
+# Source Tree
+
+The Jellyfin Source Tree is a maze of many projects. With Emby's old code and Jellyfin's new code intermixed together. To help with this, this Source Tree guide was created.
+
+### [Jellyfin Server](https://github.com/jellyfin/jellyfin)
+
+1.  BDInfo: `Blu-Ray Analyzer`
+    - Properties: `Assembly Info`
+2.  DvdLib: `DVD Anaylzer`
+    - Ifo: 
+    - Properties:
+3.  Emby.Dlna: 
+4.  Emby.Drawing:
+5.  Emby.Naming:
+6.  Emby.Notifications:
+7.  Emby.Photos:
+8.  Emby.Server.Implementations:
+9.  Jellyfin.Api:
+10. Jellyfin.Drawing.Skia:
+11. Jellyfin.Server:
+12. MediaBrowser.Api:
+    - Playback:
+      - BaseStreamingService.cs: Logic for the Stream Builder itself, this determines how transcoding will occur.
+13. MediaBrowser.Common:
+14. MediaBrowser.Controller:
+15. MediaBrowser.LocalMetadata:
+16. MediaBrowser.MediaEncoding:
+17. MediaBrowser.Model:
+18. MediaBrowser.Providers:
+19. MediaBrowser.WebDashboard:
+20. MediaBrowser.XbmcMetadata:
+21. RSSDP:
+22. benches/Jellyfin.Common.Benches:
+
+### [Web Client](https://github.com/jellyfin/jellyfin-web)
+
+1.  src: 
+    - components: `custom elements used for different sections of the user interface`
+      - playerstats:
+        - playerstats.js: `Code for displaying Playback Info during playback`
+    - controllers: `scripts that handle the logic for different pages`
+    - css:
+    - img:
+    - legacy: `currently used for all polyfills and scripts related to backwards compatibility`
+    - libraries: `dependencies that we eventually want to remove and include during the build step`
+    - scripts: `any script that isn't tied to a UI element or page but rather general functionality`
+    - splashscreens:
+    - strings: `translations for the entire interface`
+    <br>Future Implementations:
+    - assets: `images, styles, splash screens, and any other static assets`
+    - elements: `custom UI components that are used globally such as buttons or menus`
+    - themes: `all themes will exist in this folder, including custom themes at some point`
```

Comment:
The future implementations section has already replaced several of the folders that existed previously.

Reply:
Specifically, the three folders here exist now. Also, the `css`, `img`, and `splashscreens` folders are now in the `assets` folder instead.